README.md

```diff
@@ -484,6 +484,18 @@ rubocop
 
 You can read about Rubocop [here](http://rubocop.readthedocs.io/en/latest/).
 
+## Debugging with Foreman
+
+Edit `config/puma.rb` and add `worker_timeout 3600`
```

Comment:
What do you think of editing `config/puma.rb` and adding `worker_timeout 3600`?

Reply:
What about just adding this to the project?
```
worker_timeout 3600 unless Rails.env.production?
```
I think getting people to change their config will just lead to someone accidentally committing it to the project and then the timeout protection wouldn't exist in production.

Reply:
Oh true that's a better idea! Thanks!